member/index.md

```diff
@@ -16,14 +16,41 @@ We'd like you to bear in mind that none of our **moderators** work for nor are a
 #### 13. Do not attempt to take support or other requests outside of the server as we cannot ensure your or the user's safety from scams, trolling and abuse. This includes suggesting the use of DMs or remote assistance tools (such as Quick Assist or TeamViewer).
 #### 14. No content which may induce epilepsy without first making a disclaimer and obstructing the content.
 #### 15. No typing in any other language than English; we cannot moderate different languages and most people here speak in English. Failure to oblige will result in a warning or mute.
+
 ## Roles
 | Role                  | Description                                                                                                                                                                                                                                                                                                                                      |
 | --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | 🌟 Moderators         | Overlooking the community and its development; ensuring the safety of the members and the server itself.                                                                                                                                                                                                                                         |
+|   - Lead moderators                        | Be a very good moderator for a very long time and have Jaska love you |
```

Comment:
Member page shouldn't be updated, I'd prefer to have roles list short and concise here as it was, only mod page roles list should be comprehensive list.

Reply:
That's fine, I'll rollback the page then

Reply:
Ok, it rolled back to how it was,